device/mellanox/x86_64-nvidia_sn4280_simx-r0/ACS-SN4280/sai.profile

```diff
@@ -0,0 +1,4 @@
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_4280.xml
+SAI_DUMP_STORE_PATH=/var/log/mellanox/sdk-dumps
+SAI_DUMP_STORE_AMOUNT=10
```

Comment:
These two can be removed. Please check this `https://github.com/sonic-net/sonic-buildimage/pull/18074`

Reply:
done